README.md

```diff
@@ -1,5 +1,4 @@
-sync-db
-=======
+# sync-db
 
 Database synchronization utility.
```

Comment:
Command line utility to synchronize and version control relational database objects across databases.

Reply:
LGTM